includes/rest-api/class-sensei-rest-api-lesson-quiz-controller.php

```diff
@@ -251,6 +251,8 @@ private function get_quiz_meta( array $quiz_options, WP_Post $lesson ) : array {
 
 		if ( isset( $quiz_options['show_questions'] ) ) {
```

Comment:
With this change if the `show_questions` argument is not sent at all then the value will be deleted. The approach for everything else was that if an argument is omited then it is not updated and for something to get deleted `null` or `''` must be sent.

Frontend sends the value always so it makes no difference but I think that we should be consistent.

```suggestion
		if ( array_key_exists( 'show_questions', $quiz_options ) ) {
```

Reply:
The suggested change ends up setting the meta key to `NULL` because the key does exist, it's just `null`.

Reply:
That code will actually execute and set `_show_questions` to `null` when _Number of Questions_ is reset rather than clearing it.

Reply:
Yep. I have seen that we set the meta to null during importing too so I thought this would be fine. So we if `show_question` key exists but it is empty we should set the meta to empty string.

We should probably set the value to an empty string in `class-sense-import-lesson-model` line 155.